version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "1.6.0"
+	Version = "1.7.0"
```

Comment:
Why is this minor bump?

Reply:
We're not fixing a bug here, we're enabling OpenShift 4 support in the webapp by adding an env var. I think a minor bump makes more sense here.

Happy to move to a patch if you feel strongly about it though